src/SourceBuild/content/eng/arcade-overrides/SourceBuildArcade.targets

```diff
@@ -0,0 +1,100 @@
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. -->
```

Comment:
Please open a backport PR for the ArcadeOverrides and link to this PR.

Reply:
Backport of arcade overrides: https://github.com/dotnet/arcade/pull/12326